packages/core/src/sass/_globals.scss

```diff
@@ -83,37 +83,73 @@ $AU-maxwidth: 42em !default; //Answer to life, the universe, and everything (kee
 /**
  * Colors light theme
  */
-$AU-color-Text:             #313131 !default;
-$AU-color-Action:           #00698f !default;
-$AU-color-Focus:            #f04400 !default;
-$AU-color-Background-light: #ffffff !default;
+$AU-color-Text:                   #313131 !default;
+$AU-color-Action:                 #00698f !default;
+$AU-color-Focus:                  #f04400 !default;
+$AU-color-Background-light:       #ffffff !default;
+
+// Non-user definable pallet colors
+$AU-color-Muted:                  #6a6a6a;
```

Comment:
They do still need `@global` @alex-page and obviously the color-a11y function

Reply:
I'll add that to trello, my bad!